core/src/main/java/feast/core/model/EntityV2.java

```diff
@@ -60,6 +60,12 @@ public EntityV2() {
     super();
   }
 
+  /**
+   * EntityV2 object supports Entity registration outside of FeatureSet.
```

Comment:
It only applies to Feature Tables right?

Reply:
Updated.